scripts/updatekinto.js

```diff
@@ -0,0 +1,73 @@
+"use strict";
+
+const DEBUG = true;
+
+const got = require("got");
+
+const SERVER = "https://settings-writer.prod.mozaws.net/v1";
+const CID = "fxmonitor-breaches";
+/* eslint-disable-next-line no-process-env */
+const USERNAME = process.env.KINTO_USERNAME;
+/* eslint-disable-next-line no-process-env */
+const PASSWORD = process.env.KINTO_PASSWORD;
+if (!USERNAME || !PASSWORD) {
+  console.error("Please set credentials in the environment.");
+  return;
+}
+
+const AUTH = Buffer.from(`${USERNAME}:${PASSWORD}`).toString("base64");
+
+async function run() {
+  const RemoteSettingsBreachesSet = new Set((await got(
+    "https://firefox.settings.services.mozilla.com/v1/buckets/main/collections/fxmonitor-breaches/records",
```

Comment:
Wasn't sure if this should be added as a `const` above since it looks semi-similar to the `SERVER` const.

Or if we should disambiguate with a "READ_SERVER" and "WRITE_SERVER", and this may turn into:

```js
const READ_SERVER = "https://firefox.settings.services.mozilla.com/v1";
const WRITE_SERVER = "https://settings-writer.prod.mozaws.net/v1";
const CID = "fxmonitor-breaches";

//...

`${READ_SERVER}/buckets/main/collections/${CID}/records`,

//...

await got.post(`${WRITE_SERVER}/buckets/main-workspace/collections/${CID}/records`, {
```